src/app/admin/admin-campaign-add-tokens/admin-campaign-add-tokens.component.html

```diff
@@ -21,12 +21,6 @@
                 max {{ maxFundingAmount(campaignData) | currencyDefault }})
               </b>
 
-              <div class="flex flex-row gap-2 items-center mt-2 lg:mt-0 lg:justify-around">
```

Comment:
We will no longer be displaying how many tokens the admin is adding to the campaign? If that is so, then we should at least change the wording for other parts, this kind of sounds weird to me: "To raise 10$ with price per token 1$".

![Screenshot 2021-10-14 at 13 22 28](https://user-images.githubusercontent.com/12549625/137308230-7eca6cfc-58dc-4c85-b9c6-43d57ec4936e.png)

Reply:
I'll bring it back.. I had some troubles. thanks.